static/app/views/performance/traceDetails/contentV0.tsx

```diff
@@ -0,0 +1,438 @@
+import {Component, createRef, Fragment} from 'react';
+import {RouteComponentProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import {Alert} from 'sentry/components/alert';
+import GuideAnchor from 'sentry/components/assistant/guideAnchor';
+import ButtonBar from 'sentry/components/buttonBar';
+import DiscoverButton from 'sentry/components/discoverButton';
+import * as Layout from 'sentry/components/layouts/thirds';
+import ExternalLink from 'sentry/components/links/externalLink';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import TimeSince from 'sentry/components/timeSince';
+import {t, tct, tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Organization} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import EventView from 'sentry/utils/discover/eventView';
+import {QueryError} from 'sentry/utils/discover/genericDiscoverQuery';
+import {getDuration} from 'sentry/utils/formatters';
+import {createFuzzySearch, Fuse} from 'sentry/utils/fuzzySearch';
+import getDynamicText from 'sentry/utils/getDynamicText';
+import {
+  TraceError,
+  TraceFullDetailed,
+  TraceMeta,
+} from 'sentry/utils/performance/quickTrace/types';
+import {filterTrace, reduceTrace} from 'sentry/utils/performance/quickTrace/utils';
+import {VisuallyCompleteWithData} from 'sentry/utils/performanceForSentry';
+import Breadcrumb from 'sentry/views/performance/breadcrumb';
+import {MetaData} from 'sentry/views/performance/transactionDetails/styles';
+
+import {TraceDetailHeader, TraceSearchBar, TraceSearchContainer} from './styles';
+import TraceNotFound from './traceNotFound';
+import TraceView from './traceView';
+import {TraceInfo} from './types';
+import {getTraceInfo, hasTraceData, isRootTransaction} from './utils';
+
+type IndexedFusedTransaction = {
+  event: TraceFullDetailed | TraceError;
+  indexed: string[];
+};
+
+type Props = Pick<RouteComponentProps<{traceSlug: string}, {}>, 'params' | 'location'> & {
+  dateSelected: boolean;
+  error: QueryError | null;
+  isLoading: boolean;
+  meta: TraceMeta | null;
+  organization: Organization;
+  traceEventView: EventView;
+  traceSlug: string;
+  traces: TraceFullDetailed[] | null;
+  handleLimitChange?: (newLimit: number) => void;
+  orphanErrors?: TraceError[];
+};
+
+type State = {
+  filteredEventIds: Set<string> | undefined;
+  searchQuery: string | undefined;
+};
+
+class TraceDetailsContentV0 extends Component<Props, State> {
```

Comment:
Added a seperate file for this component as we will port over event detail data like:
1. Tags
2. Web vitals
3. Custom Performance Metrics
To it.